msbuild/Xamarin.Mac.Tasks.Core/Tasks/DetectSdkLocationsTaskBase.cs

```diff
@@ -93,18 +93,25 @@ void EnsureSdkPath ()
 				Log.LogError ("Could not locate SDK bin directory");
 		}
 
-		void EnsureAppleSdkRoot ()
+		bool EnsureAppleSdkRoot ()
 		{
 			if (!MacOSXSdks.Native.IsInstalled) {
-				Log.LogError ("Could not find a valid Xcode app bundle");
-			} else {
-				Log.LogMessage (MessageImportance.Low, "DeveloperRoot: {0}", MacOSXSdks.Native.DeveloperRoot);
-				Log.LogMessage (MessageImportance.Low, "GetPlatformPath: {0}", MacOSXSdks.Native.GetPlatformPath ());
-
-				SdkDevPath = MacOSXSdks.Native.DeveloperRoot;
-				if (string.IsNullOrEmpty (SdkDevPath))
-					Log.LogError ("Could not find a valid Xcode developer path");
+				var ideSdkPath = "(Project > SDK Locations > Apple > Apple SDK)";
+#if WINDOWS
+				var ideSdkPath = "(Tools > Options > Xamarin > iOS Settings > Apple SDK)";
```

Comment:
I've just notice the base tasks are being built only on the Mac, we get those from the bundle.zip, so the Windows check should be on runtime.